app/controllers/admin/site_options_controller.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module Admin
+  class SiteOptionsController < AdminController
+    load_and_authorize_resource class: SiteOption
+
+    def update
+      option_keys = params.keys.select { |key| SiteOption::OPTIONS.include?(key) }
+      option_keys.each do |option_key|
+        option = SiteOption.find_by(name: option_key)
```

Comment:
Here's one possible way to handle the thing we talked about (as an alternative to a db seeds update):
```suggestion
        option = SiteOption.find_by(name: option_key)
        option = SiteOption.create!(name: option_key, default_value_for_option(option_key))
```

And then you could add a default_value_for_option method that handles various defaults (e.g. `return 1 if option_key == 'deposits_enabled'`).